### PR TITLE
disable the "release" stage if forked repo

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

**Comment by xuwei-k** [comment]
>  this is needed in the following situation

Yes, and also push master branch

**Comment by olafurpg** [comment]
Is there a way to make the conditional work for any fork without manually typing organization/repo-name? I'd like to reduce as many manual steps as possible in the installation instructions. Most people copy-paste from the docs without too much thought.

Otherwise, I think it's fair to expect forks that enable Travis CI to update `.travis.yml` to remove the `release` stage.

**Comment by olafurpg** [comment]
Alternatively, if there is an environment variable that we can detect for forks then we can update the plugin implementation to do nothing on `sbt ci-release`. I'm fine with the change itself, I'm only concerned about pushing further complexity into `.travis.yml`.

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jdk:
 stages:
   - name: test
   - name: release
-    if: (branch = master AND type = push) OR (tag IS present)
+    if: ((branch = master AND type = push) OR (tag IS present)) AND (repo = olafurpg/sbt-ci-release)
```
Comment (gabro) [inline .travis.yml]:
```suggestion
    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

File: readme.md [hunk 2]
```diff
@@ -201,7 +201,7 @@ before_install:
 stages:
   - name: test
   - name: release
-    if: (branch = master AND type = push) OR (tag IS present)
+    if: ((branch = master AND type = push) OR (tag IS present)) AND (repo = your-github-id-or-organization-id/your-repo-name)
```
Comment (gabro) [inline readme.md]:
```suggestion
    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

**Comment by olafurpg** [comment]
Discovered by @gabro, the condition might be simplified to `&& !fork` https://docs.travis-ci.com/user/conditions-v1#integration

@xuwei-k could you confirm this setting works as expected? In that case, I'm happy to merge a PR adding `&& !fork` to the installation instructions.

**Comment by olafurpg** [comment]
Guess we can merge and see if master still releases. LGTM!

**Comment by olafurpg** [comment]
Looks like there's still a "Release" stage https://travis-ci.org/olafurpg/sbt-ci-release/builds/523378581 :tada:

**Pull request merged**